spec/controllers/application_controller_spec.rb

```diff
@@ -31,16 +31,10 @@ def missing_template
   end
 
   context "when a request raises an ActionView::MissingTemplate error" do
-    context "when the format for the request is a valid format" do
-      it "should raise an error" do
-        expect { get :missing_template, :format => 'json' }.to raise_error(ActionView::MissingTemplate)
-      end
-    end
-
     context "when the format for the request is not a valid format" do
-      it "should render a 406" do
+      it "should render a 204" do
```

Comment:
We could clean this up by using single instead of double quotes throughout this spec, and by omitting the `should` from the spec description:
```ruby
it 'responds with a 204' do
```